WordPress/Classes/Stores/StatsInsightsStore.swift

```diff
@@ -120,6 +120,8 @@ struct InsightStoreState {
 class StatsInsightsStore: QueryStore<InsightStoreState, InsightQuery> {
 
     fileprivate static let cacheTTL: TimeInterval = 300 // 5 minutes
+    private let cache: StatsInsightsCache = .shared
+    private var lastRefreshDates: [NSNumber: Date] = [:]
```

Comment:
Probably out of scope for this PR, but I'm wondering if we should move the "cache expiry check" (last refreshing time and TTL) to the cache type, so that most of the cache related stuff is encapsulated in one place.